libc-test/semver/hurd.txt

```diff
@@ -0,0 +1 @@
+getdents64
```

Comment:
There are exceptions but in general semver files for T3 targets don't get checked, so this can be dropped